content/open-source/oci/what-are-oci-artifacts.md

```diff
@@ -0,0 +1,53 @@
+“config”: {
+“mediaType”: “application/vnd.yourcustomartifact+json”,
+“Size”: 233,
+“Digest”: “sha245:...”
+},
+“Layers”: [
+{
+“mediaType:” “application/vnd.yourcustomartifact.tar.gzip”,
+“Size”: 680,
+“Digest”: “sha245:...”
+}
+]
+}
+```
+
+In the example manifest above, the  `config` field contains the custom `mediaType` value `application/vnd.yourcustomartifact+json` and the `layers` fields contains the custom `mediaType` value `application/vnd.yourcustomartifact.tar.gzip`.
+
+
+## OCI Artifacts and Software Supply Chain Security 
+For software supply chain security, OCI artifacts offer a useful way to store [SBOMs](​​https://edu.chainguard.dev/open-source/sbom/what-is-an-sbom/) and signatures inside a container registry. 
+
+An **SBOM**, or software bill of materials, is a formally structured list of libraries, modules, licensing, and version information that make up any given piece of software. When a vulnerability security advisory is issued, SBOMs enable software operators to quickly see whether their codebase contains any components associated with the vulnerability. A [**signature**](https://edu.chainguard.dev/open-source/sigstore/cosign/an-introduction-to-cosign/) is a way of verifying that you are the author of your software, and that it has not been tampered with by a third party. 
+
+The open source tool [Cosign](https://github.com/sigstore/cosign), part of the [Sigstore](https://www.sigstore.dev/) project, enables container engineers to sign their images and artifacts and store them as an artifact in an OCI registry. You can also use Cosign to store SBOMs as artifacts in an OCI registry. To learn more about storing signatures as artifacts, visit the [section on counter signing](
+https://github.com/sigstore/cosign#counter-signing) in the Cosign repo. To learn more about storing SBOMs as artifacts, visit the [section on signing SBOMs](https://docs.sigstore.dev/cosign/other_types/#sboms-software-bill-of-materials) in Sigstore’s documentation.  
+
+ ## Considerations 
+Community usage and guidance of OCI artifacts is actively evolving and there are a few things to keep in mind when using them. First, not all registries support artifacts, and the [OCI Image Specification](https://github.com/opencontainers/image-spec) recommends avoiding the use of artifacts if you are concerned about portability. Best practices are still also under debate, giving rise to the [OCI Reference Types Working Group](https://github.com/opencontainers/wg-reference-types)], which is considering different ways of describing and handling objects stored in an OCI registry.
+
+To learn more about OCI artifacts, visit the Open Container Initiative’s [artifact repo on Github](https://github.com/opencontainers/artifacts). You can also read about the current proposals for the OCI Reference Types Working Groups by visiting the [Intro to OCI Reference Types](https://www.chainguard.dev/unchained/intro-to-oci-reference-types) post on Chainguard’s blog. 
```

Comment:
I think we still plan to archive the artifacts repo and move its guidance into the image-spec repo, so maybe just drop this link for now, since it might get stale.